sources/packages/web/src/views/institution/Login.vue

```diff
@@ -1,49 +1,74 @@
-        The user was validated successfully but is not currently allowed to have
-        access to this application. Please contact the Administrator for more
-        information
-      </Message>
-    </v-card-text>
-    <v-card-actions>
-      <v-row justify="center" class="my-3">
-        <v-btn
-          class="primary-btn-background"
-          @click="login"
-          prepend-icon="fa:fa fa-user"
-        >
-          Login with BCeID
-        </v-btn>
-        <v-btn
-          class="primary-btn-background"
-          @click="login"
-          variant="outlined"
-          prepend-icon="fa:fa fa-user-plus"
-        >
-          Sign Up with BCeID
-        </v-btn>
+  <v-card elevation="2" class="mx-auto mt-12" max-width="620px" outlined>
+    <v-card-text>
+      <v-row no-gutters>
+        <v-col cols="9">
+          <h1 class="category-header-large primary-color">
+            Welcome to StudentAid BC
+          </h1>
+          <p class="mb-5">
+            Login or sign up here to manage your institution account.
+          </p>
+          <content-group>
+            <v-row>
+              <v-col>
+                <h3 class="category-header-medium primary-color">
+                  Login with BCeID
+                </h3>
+                <p class="sign-in-description">
+                  For returning users—login using your BCeID.
+                </p>
+                <v-btn
+                  class="primary-btn-background"
+                  @click="login"
+                  prepend-icon="fa:fa fa-user"
+                >
+                  Login with BCeID
+                </v-btn>
+              </v-col>
+              <v-col
+                ><h3 class="category-header-medium primary-color">
+                  Sign up with BCeID
+                </h3>
+                <p class="sign-in-description">
+                  For new users—sign up using your BCeID.
+                </p>
+                <v-btn
+                  class="primary-btn-background"
+                  @click="login"
+                  variant="outlined"
+                  prepend-icon="fa:fa fa-user-plus"
+                >
+                  Sign Up with BCeID
+                </v-btn></v-col
+              >
+            </v-row>
+          </content-group>
+        </v-col>
+        <v-col
+          ><v-img
+            height="260"
+            class="ml-2"
+            alt="Person standing with laptop"
+            src="@/assets/images/person-standing-with-laptop.svg"
+        /></v-col>
+        <v-col cols="12" class="my-n3">
+          <Message severity="error" v-if="showBasicBCeIDMessage">
+            No such Business account has been found with BCeID. Please login
+            with your Business BCeId
+          </Message>
+          <Message severity="error" v-if="showDisabledUserMessage">
+            Disabled User - you don't have access to the system. Please contact
+            Administrator for more informations.
```

Comment:
Please change "Administrator for more informations.", to "Administrator for more information."